nucypher/blockchain/eth/token.py

```diff
@@ -833,6 +834,9 @@ def __read_stakes(self) -> None:
                 if onchain_stake.final_locked_period > terminal_period:
                     terminal_period = onchain_stake.final_locked_period
 
+                if onchain_stake.status() is Stake.Status.INACTIVE:
```

Comment:
```suggestion
                if onchain_stake.status().is_child(Stake.Status.INACTIVE):
```
I know that both ways are equal but I see suggestion as more failsafe way because if we change anything in status this will apply automatically to `is_child`

Reply:
👍